core/src/main/java/tc/oc/pgm/blockdrops/BlockDropsModule.java

```diff
@@ -53,6 +54,8 @@ public BlockDropsModule parse(MapFactory factory, Logger logger, Document doc)
       List<BlockDropsRule> rules = new ArrayList<>();
       FilterParser filterParser = factory.getFilters();
       RegionParser regionParser = factory.getRegions();
+      final Optional<ItemModifyModule> itemModifier =
+          Optional.ofNullable(factory.getModule(ItemModifyModule.class));
```

Comment:
You probably need a dependency for this, or depending on the map you may sometimes see this work or not